charts/consul/values.yaml

```diff
@@ -1925,6 +1925,17 @@ dns:
     # Port number to be used by DNS proxy
     port: 53
 
+    # Refers to a Kubernetes secret that you have created that contains
+    # an ACL token for your Consul cluster which allows the dns proxy the correct
+    # permissions. This is only needed if ACLs are managed manually within the Consul cluster, i.e. `global.acls.manageSystemACLs` is `false`.
```

Comment:
Maybe we can add what are the correct permissions that dns-proxy needs in that case?